docusaurus/docs/Flutter/guides/adding_localization.mdx

```diff
@@ -79,6 +82,43 @@ To override an existing language, create a new class extending that particular l
 
 Checkout [this example](https://github.com/GetStream/stream-chat-flutter/blob/master/packages/stream_chat_localizations/example/lib/override_lang.dart) to see how to override an existing language.
 
+### Changing the default language
+
+To change the default language you can use the `MaterialApp.localeListResolutionCallback` property.
+Here is an example of how that would look like:
+
+```dart
+    MaterialApp(
+        theme: ThemeData.light(),
+        darkTheme: ThemeData.dark(),
+        // Add all the supported locales
+        supportedLocales: const [
+            Locale('en'),
+            Locale('hi'),
+            Locale('fr'),
+            Locale('it'),
+        ],
+        localeListResolutionCallback: (locales, supportedLocales) {
+            final supportedLanguageCodes =
+                supportedLocales.map((e) => e.languageCode);
+            if (locales != null) {
+            for (final locale in locales) {
+                if (supportedLanguageCodes.contains(locale.languageCode)) {
+                return locale;
+                }
+            }
+            }
+
+            return const Locale('it');
```

Comment:
some formatting issues, we can also add doc comments mentioning what we are doing